frontend/src/state-providers/fonts.ts

```diff
@@ -58,14 +64,13 @@ export function createFontsState(editor: Editor) {
 					const files = new Map(font.variants.map((x) => [formatFontStyleName(x), font.files[x]]));
 					return { family, variants, files };
 				});
-				state.fontNames = result.map((value) => value.family);
 
 				resolve(result);
 			});
 	});
 
 	return {
-		state: readonly(state) as typeof state,
+		fontNames,
```

Comment:
If we aren't returning a state, I think this file should be moved out of `/state-providers`, maybe restructured to fit within either `/io-managers` or `/utility-functions`?